src/index.ts

```diff
@@ -246,8 +253,8 @@ export function startWallProfiling(): void {
       .profile({
         lineNumbers: true,
         sourceMapper: config.sm,
-        durationMillis: INTERVAL,
-        intervalMicros: 10000,
+        durationMillis: 1000 || SAMPLING_DURATION,
+        intervalMicros: 10000 || SAMPLE_RATE,
```

Comment:
Given that we already have defined default values, should we maybe just use SAMPLING_DURATION and SAMPLE_RATE here and in other places without any defaults?

Reply:
Absolutely! I will correct this.